service/src/main/resources/application.yml

```diff
@@ -0,0 +1,46 @@
+logging:
+  level.bio.terra.batchanalysis: ${BATCHANALYSIS_LOG_LEVEL:debug}
+  pattern.level: '%X{requestId} %5p'
+
+server:
+  compression:
+    enabled: true
+    mime-types: text/css,application/javascript
+  port: 8080
+
+spring:
+  # application name and version are used to populate the logging serviceContext
+  # https://github.com/DataBiosphere/terra-common-lib/blob/480ab3daae282ddff0fef8dc329494a4422e32f1/src/main/java/bio/terra/common/logging/GoogleJsonLayout.java#L118
+  application.name: cbas
+  application.version: ${cbas.version.gitHash:unknown}
+
+  config.import: optional:classpath:rendered/providers.yaml;classpath:rendered/version.properties;optional:classpath:rendered/allowed_jwks.yaml;optional:classpath:rendered/allowed_jwt_issuers.yaml
```

Comment:
In catalog we aren't using this (although it looks like there's still some cruft in catalog from the copied config) so I would bet that you don't need it for this project.

Reply:
I'm going to follow the "remove it as long as it doesn't break" rule. The less "cruft that we might need one day", the better